jetty-websocket/websocket-common/src/main/java/org/eclipse/jetty/websocket/common/extensions/compress/CompressExtension.java

```diff
@@ -429,6 +429,7 @@ public String toString()
         @Override
         public void failed(Throwable x)
         {
+            notifyCallbackFailure(current.callback, x);
```

Comment:
Seems like this is called twice now when `writeFailed` is called, isn't it?

Reply:
true, I think this one should replace the call to `notifyCallbackFailure()` in `writeFailed()` which will call `failed()` anyway

Reply:
Exactly thinking the same